server_admin/topics/events/login.adoc

```diff
@@ -1,70 +1,111 @@
 
 === Login Events
 
-Login events occur for things like when a user logs in successfully, when somebody enters in a bad password, or when a user account
-is updated.  Every single event that happens to a user can be recorded and viewed.  By default, no events are stored
-or viewed in the Admin Console.  Only error events are logged to the console and the server's log file.  To start
-persisting  you'll need to enable storage.  Go to the `Events` left menu item and select the `Config` tab.
+{project_name} triggers a login event for actions such as successful user login, a user entering an incorrect password, or a user account updating. {project_name} can record and make available to view every single event that affects users. By default, {project_name} does not store or display events in the Admin Console. {project_name} logs error events to the console and the server's log file only.
```

Comment:
Four uses of {project_name} in one paragraph.  Better to focus on the user, not the product. How is this
"You can record and view every event that affects users. {project_name} triggers login events for actions such as successful user login, a user entering an incorrect password, or a user account updating. By default, {project_name} does not store or display events in the Admin Console. Only the error events are logged to the Admin Console and the server’s log file."